integration/common/openlineage/common/provider/dbt/structured_logs.py

```diff
@@ -424,6 +464,9 @@ def _get_job_name(self, event) -> str:
         elif node_unique_id.startswith("snapshot"):
```

Comment:
I repeated what was done in the `processor.py`. IMO this should be as simple as the `node_id` it's unique across all dbt nodes of the project.

Same as for Airflow task for example https://openlineage.io/docs/spec/naming/#job-naming

LMK what you think !